### ISSUE TITLE
Error Getting 500 Internal Server Error

### ISSUE THREAD

**Issue by sachinparmartot** [akeneo/pim-community-dev#15479]
Hello Team,

I am trying to install akeneo PIM community edition in my Ubuntu Server
my server is in AWS with 4v core CPU and 16 Gb Ram
while i try to install akeneo  i am getting bellow error.

Oops! An Error Occurred
The server returned a "500 Internal Server Error".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.

i have try to many time to install i have follow all steps as per official sites. here i have attach image please check and update me

i am waitting for your response

![akeneo](https://user-images.githubusercontent.com/92729524/137729326-c652e90d-4f98-4ed8-9396-d2748412e5fe.jpg)

**Comment by sachinparmartot** [comment]
![500 internal](https://user-images.githubusercontent.com/92729524/137729574-8bc7becf-63db-4b3b-aa00-42e50b7aaaed.jpg)

While i check this site it will getting above error.

**Comment by krishankantshandilya** [comment]
Hi @centoyas ,

Have you created a DB for Akeneo and added a valid database connection to the .env file?

There seems to be a problem when the Akeneo installation command is trying to connect to the database.

**Comment by centoyas** [comment]
Wow, its been a long time ago.. i dont remeber , i can try again and tell you but i remeber i had to cp my env file cause the script replaced it

**Comment by krishankantshandilya** [comment]
Yes, when we run the command NO_DOCKER=true make dev then it resets the env files so before running this command you should cp the env file and then before running the command check that the env file variables reset or not. If the variables are reset right after running the command then paste the contents of the copied env file data in the .env file between command is running.

**Comment by centoyas** [comment]
Yep dat was exactly what i did
